### PR TITLE
Audio PR - Augmentation support [ Mel-Filter Bank and Normalize ]

### PR DESCRIPTION
**Pull request by SundarRajan98**
Adding openvx rpp extensions for Mel-Filter bank and Normalize augmentations
Note: PR #1324 has to be merged before this PR - file changes will reduce to 9

### CODE REVIEW

File: amd_openvx_extensions/amd_rpp/source/tensor/MelFilterBank.cpp
Comment (rrawther) [inline amd_openvx_extensions/amd_rpp/source/tensor/MelFilterBank.cpp:121]: follow the same comment in other node

File: amd_openvx_extensions/amd_rpp/source/tensor/Normalize.cpp
Comment (rrawther) [inline amd_openvx_extensions/amd_rpp/source/tensor/Normalize.cpp:73]: It would be better to pass a pointer to RPP for these pointers instead of reformatting the data

**Review by rrawther — CHANGES_REQUESTED** [review]
Added couple of comments

File: amd_openvx_extensions/amd_rpp/source/tensor/Normalize.cpp
Comment (SundarRajan28) [reply amd_openvx_extensions/amd_rpp/source/tensor/Normalize.cpp:71]: The ROI data for audio tensors is not compatible with the normalize kernel requirements. We are reformatting the data to pass the ROI in the required format for normalize kernel

Comment (rrawther) [reply amd_openvx_extensions/amd_rpp/source/tensor/Normalize.cpp:71]: I know that with current implementation. But I have mentioned this many times before. We want to have the vx_rpp layer thin and transparent. I still see the same old style is followed

**Comment by LakshmiKumar23** [comment]
@swetha097 Is this the last PR for Audio for MIVisionX? If so, the patch version in vx_rpp_ext_version file needs to be changed. Else rocAL cannot recognize it going forward

File: amd_openvx_extensions/amd_rpp/source/tensor/NonSilentRegionDetection.cpp
Comment (rrawther) [inline amd_openvx_extensions/amd_rpp/source/tensor/NonSilentRegionDetection.cpp:225]: remove this label if not used

File: amd_openvx_extensions/amd_rpp/source/tensor/Normalize.cpp
Comment (rrawther) [inline amd_openvx_extensions/amd_rpp/source/tensor/Normalize.cpp:238]: check data->pMean validity

Comment (rrawther) [reply amd_openvx_extensions/amd_rpp/source/tensor/Normalize.cpp:238]: same here as above

File: amd_openvx_extensions/amd_rpp/source/tensor/Normalize.cpp
Comment (rrawther) [inline amd_openvx_extensions/amd_rpp/source/tensor/Normalize.cpp:241]: same here as above

File: amd_openvx_extensions/amd_rpp/source/tensor/Normalize.cpp
Comment (rrawther) [inline amd_openvx_extensions/amd_rpp/source/tensor/Normalize.cpp:313]: remove this

File: amd_openvx_extensions/amd_rpp/source/tensor/Slice.cpp
Comment (rrawther) [inline amd_openvx_extensions/amd_rpp/source/tensor/Slice.cpp:298]: remove this

**Review by rrawther — CHANGES_REQUESTED** [review]
please address review comments

File: amd_openvx_extensions/amd_rpp/source/tensor/NonSilentRegionDetection.cpp
Comment (swetha097) [reply amd_openvx_extensions/amd_rpp/source/tensor/NonSilentRegionDetection.cpp:225]: This label seems to be used as a part of the internal_rpp.h file. Gives error if removed from the files

File: amd_openvx_extensions/amd_rpp/source/tensor/Slice.cpp
Comment (swetha097) [reply amd_openvx_extensions/amd_rpp/source/tensor/Slice.cpp:298]: This label seems to be used as a part of the internal_rpp.h file. Gives error if removed from the files

File: amd_openvx_extensions/amd_rpp/source/tensor/Normalize.cpp
Comment (swetha097) [reply amd_openvx_extensions/amd_rpp/source/tensor/Normalize.cpp:315]: This label seems to be used as a part of the internal_rpp.h file. Gives error if removed from the files

File: amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp
Comment (LakshmiKumar23) [inline amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp:2878]: same as previous PR. Redundant checks

File: amd_openvx_extensions/amd_rpp/source/tensor/Normalize.cpp
Comment (swetha097) [reply amd_openvx_extensions/amd_rpp/source/tensor/Normalize.cpp]: Done

File: amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp
Comment (swetha097) [reply amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp]: Merged with prev PR, this issue is resolved in the prev PR

**Comment by swetha097** [comment]
> the patch version in vx_rpp_ext_version file

All comments have been addressed except patch version in vx_rpp_ext_version file.

File: amd_openvx_extensions/amd_rpp/source/tensor/Normalize.cpp
Comment (swetha097) [reply amd_openvx_extensions/amd_rpp/source/tensor/Normalize.cpp]: Addressed

**Comment by kiritigowda** [comment]
@SundarRajan98 can you please sync to TOT

**Review by rrawther — APPROVED** [review]

**Comment by LakshmiKumar23** [comment]
> > the patch version in vx_rpp_ext_version file
> 
> All comments have been addressed except patch version updation in vx_rpp_ext_version file.

I can add this as a separate PR once this is merged

**Comment by kiritigowda** [comment]
@SundarRajan98 can you fix merge conflicts

**Review by LakshmiKumar23 — APPROVED** [review]
Can be merged after syncing with ToT

**Pull request merged**
